### ISSUE TITLE
propagate error in two consecutive CNN

### ISSUE THREAD

**Issue by nadianaji** [keras-team/keras#10241]
Hi,
I have a structure that contains two consecutive CNN and the output of one network is used as an input to next network. now I do not know how should I use loss function. I want to use MSE for second network but I need to consider the effect of second network on first network and the loss function of first network should be consider the error of second network. Can you please help me about loss functions when I have a chain of CNN and the output of one is the input of next CNN?
